Python/Product/PythonTools/PythonTools/Options/ExperimentalOptions.cs

```diff
@@ -61,5 +64,11 @@ public sealed class ExperimentalOptions {
         /// True to use conda for package management when available.
         /// </summary>
         public bool UseCondaPackageManager { get; set; }
+
+        /// <summary>
+        /// True to use new Ptvs debugger built to run on VS Code Debug Adapter Host
```

Comment:
Always "PTVS"

Reply:
And rarely needs to be named within our own project - this could just read "Use a debugger built to run on VS Code Debug Adapter Host"